src/components/NewService.vue

```diff
@@ -59,30 +92,22 @@
       <p class="title">
         Docker Service ID
       </p>
+      <p class="subtitle">
+        The ID of the unmanaged Service within Docker (e.g. the name of the stack in Docker Swarm)
```

Comment:
This may not so clear to the readers. Say you have orion deployed in a remote cluster, you want to register it as unmanaged in your cluster, what's that you put here? Is the stack name just an arbitrary example (e.g) or something required (i.e)?